ui/hooks/useSafeGasEstimatePolling.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+import {
+  disconnectGasFeeEstimatePoller,
+  getGasFeeEstimatesAndStartPolling,
+} from '../store/actions';
+
+export function useSafeGasEstimatePolling() {
```

Comment:
add some documentation

Reply:
- [ ] Add some documentaiton

This is handling the case describe in this comment on a separate PR: https://github.com/MetaMask/metamask-extension/pull/11511/files#r670686360